dali/pipeline/operator/sequence_operator.h

```diff
@@ -55,12 +110,13 @@ inline bool is_per_frame(const TensorVector<CPUBackend> &arg_input) {
  * resize the outputs.
  */
 template <typename Backend>
-class SequenceOperator : public Operator<Backend> {
+class SequenceOperator : public Operator<Backend>, public SampleBroadcasting<Backend> {
```

Comment:
Wouldn't this suffice?
```suggestion
class SequenceOperator : public Operator<Backend>, protected SampleBroadcasting<Backend> {
```
You're re-exposing the function BroadcastSamples anyway.